src/utils/counter.ts

```diff
@@ -0,0 +1,15 @@
+const counter = new Map();
+
+export const increaseCount = (url: string): void => {
+  if (counter.has(url)) {
+    counter.set(url, 1);
```

Comment:
It seems like we have to invert this condition, so that if `counter` has `url` it means that we have to increase the count, but not to set it into `1`.